.github/workflows/site_lints.yml

```diff
@@ -0,0 +1,28 @@
+name: Site lints
+
+on:
+  push:
+    paths:
+      - site/**/*.tsx
+      - site/**/*.ts
+      - site/yarn.lock
+
+jobs:
+  eslint:
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+
+      - name: Install dependencies
+        run: npm install
```

Comment:
Might be better to use yarn, as that's what the lock uses

Reply:
Done!